src/tic_tac_toe/board.clj

```diff
@@ -0,0 +1,18 @@
+(ns tic-tac-toe.board
+  (:use [clojure.math.numeric-tower :only [expt]]))
+
+(defn make
+  [size]
+  (vec (repeat (expt size 2) " ")))
```

Comment:
Is there something about `(expt size 2)` that's more efficient than `(* size size)`

Reply:
I hope that didn't come off as snarky, I was just wondering what prompted importing the exponent function instead of using `*`

Reply:
Nope.  I would rather just use `(* size size)`, so I'm going to change it.

Reply:
[It's gone.](https://github.com/saralein/tic-tac-toe_clojure/pull/1/files#diff-f87562537f8b226db0bb61de2b7a7fe8R3).